notifications_utils/recipients.py

```diff
@@ -420,7 +428,7 @@ def try_validate_and_format_phone_number(number, column=None, international=None
         return validate_and_format_phone_number(number, column, international)
     except InvalidPhoneError as exc:
         if log_msg:
-            current_app.logger.warning('{}: {}'.format(log_msg, exc))
+            logging.warning('{}: {}'.format(log_msg, exc))
```

Comment:
https://github.com/department-of-veterans-affairs/vanotify-team/blob/master/Engineering/logging_policy.md#style

Reply:
`logging.warning("%s: %s", log_msg, exc)`

Reply:
Perhaps better:
```
logging.warning(log_msg)
logging.exception(exc)
```
@k-macmillan What do you think?  "exc" is an exception.

Reply:
That's fine.